server/uptime-kuma-server.js

```diff
@@ -186,7 +186,8 @@ class UptimeKumaServer {
     }
 
     async getClientIP(socket) {
-        let clientIP = socket.client.conn.remoteAddress;
+        // Check if cloudflare connecting IP for real IP in proxy tunnels
+        let clientIP = socket.client.conn.request.headers["cf-connecting-ip"] || socket.client.conn.remoteAddress;
```

Comment:
Security issue, this header is not trusted.

Similar to x-forwarded-for and x-real-ip, it is trusted only if trustProxy is true.